srcpkgs/freetype/template

```diff
@@ -16,10 +16,16 @@ checksum=86a854d8905b19698bbc8f23b860bc104246ce4854dcea8e3b0fb21284f75784
 post_install() {
 	vlicense docs/LICENSE.TXT
 	vlicense docs/FTL.TXT
+	if [ "$CROSS_BUILD" ]; then
+		sed -i -e "s/${XBPS_CROSS_TRIPLET}-pkg-config/pkg-config/" \
```

Comment:
`vsed`?

Reply:
I think `sed` is fine in `post_install`. :shrug:

Reply:
I ended up `sed` in `post_patch` instead.